popSimManu.tex

```diff
@@ -599,7 +628,12 @@ \section*{Discussion}
 (Figure~\ref{fig:IM_popn_human}), most methods do reasonably well at
 reconstructing the YRI history, but struggle with the more complex CEU
 history, in large part because of the restriction of constant sizes per
-population.  An exception is \smcpp, which does not have the same
+population.  
+% ALG: I would rephrase these to make clear it’s the simulated history.
+% Otherwise, it sounds like you are saying that CEU have a more complicated 
+% history than YRI… which I doubt is true, it’s just that in our models they 
+% have a more complicated history. 
```

Comment:
I am restraining myself from saying "drastically oversimplified YRI history", since it's YRI that really has the more complex history... =)

Reply:
yeah...

Reply:
I've added "simulated" in front of YRI and CEU.